modules/microshift-low-latency-rhel-edge-blueprint-rtk.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="microshift-low-latency-blueprint-rhel-edge-rtk_{context}"]
+= Reference blueprint for installing {op-system-rt-kernel} in a {op-system-ostree} image
+
+//TODO add footnotes for each group; check install doc blueprint and see if too much duplication
+
+.Example blueprint used to install the real-time kernel in a {op-system-ostree} image
++
+[source,text]
+----
+name = "microshift-low-latency"
+description = "RHEL 9.4 and MicroShift configured for low latency"
+version = "0.0.1"
+modules = []
+groups = []
+distro = "rhel-94"
+
+[[packages]]
+name = "microshift"
+version = "*"
+
+[[packages]]
+name = "microshift-greenboot"
+version = "*"
+
+[[packages]]
+name = "microshift-networking"
+version = "*"
+
+[[packages]]
+name = "microshift-selinux"
+version = "*"
```

Comment:
```suggestion
```

Reply:
This is the reference module showing everything. It's at the end and the user can skip it, unless not so experienced, or just wants to see the whole thing.